app/Http/Controllers/User/BookController.php

```diff
@@ -134,6 +140,20 @@ public function store(BookRequest $request)
                 'book_id' => $book->id,
             ];
             $this->owner->store($data);
+            
+            //create information for follower 
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Controllers/User/BookController.php</br>**Line**: 144</br>Whitespace found at end of line